Description: Search for gnutls with pkg-config. Required for gnutls 2.8.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Bug-Debian: http://bugs.debian.org/529915
Last-Update: 2014-08-09

--- wmbiff-0.4.27.orig/configure.ac
+++ wmbiff-0.4.27/configure.ac
@@ -77,13 +77,22 @@ AC_ARG_ENABLE(crypto, AC_HELP_STRING([ -
 
 GNUTLS_MAN_STATUS="This copy of WMBiff was not compiled with GNUTLS."
 if test "$gnutls" = "ok"; then
- AM_PATH_LIBGNUTLS(1.0.4, [LIBS="$LIBS $LIBGNUTLS_LIBS"
-                          CFLAGS="$CFLAGS $LIBGNUTLS_CFLAGS"
-                         CPPFLAGS="$CPPFLAGS $LIBGNUTLS_CFLAGS"
-                         GNUTLS_COMMON_O="gnutls-common.o"
-                         GNUTLS_MAN_STATUS="This copy of WMBiff was compiled with GNUTLS."
-                         AC_CHECK_HEADERS(gnutls/gnutls.h) ],
-                         [ echo GNUTLS can be found at ftp://gnutls.hellug.gr/pub/gnutls ])
+
+PKG_CHECK_MODULES([LIBGNUTLS], [gnutls >= 1.2.0],
+	[LIBS="$LIBS $LIBGNUTLS_LIBS"
+        CFLAGS="$CFLAGS $LIBGNUTLS_CFLAGS"
+        CPPFLAGS="$CPPFLAGS $LIBGNUTLS_CFLAGS"
+        GNUTLS_COMMON_O="gnutls-common.o"
+        GNUTLS_MAN_STATUS="This copy of WMBiff was compiled with GNUTLS."
+        AC_CHECK_HEADERS(gnutls/gnutls.h) ],
+        [ echo GNUTLS can be found at ftp://gnutls.hellug.gr/pub/gnutls ])
+## AM_PATH_LIBGNUTLS(1.0.4, [LIBS="$LIBS $LIBGNUTLS_LIBS"
+##                          CFLAGS="$CFLAGS $LIBGNUTLS_CFLAGS"
+##                         CPPFLAGS="$CPPFLAGS $LIBGNUTLS_CFLAGS"
+##                         GNUTLS_COMMON_O="gnutls-common.o"
+##                         GNUTLS_MAN_STATUS="This copy of WMBiff was compiled with GNUTLS."
+##                         AC_CHECK_HEADERS(gnutls/gnutls.h) ],
+##                         [ echo GNUTLS can be found at ftp://gnutls.hellug.gr/pub/gnutls ])
 else
  AC_MSG_RESULT(GNUTLS support requires libz.a and libgdbm.a, so will be disabled)
 fi
